.rubocop.yml

```diff
@@ -81,3 +81,40 @@ Style/AccessorMethodName:
 Style/TrivialAccessors:
   Enabled: false
 
+# All the configuration below this comment tries to avoid offenses because of backported
```

Comment:
because of backported code :)

Not really, these are rules and files that were failing, so just exclude them.